### DISCUSSION TITLE
bininninjia commercial latest version save AwemeCore.bndb crashes on completion of analysis

### DISCUSSION THREAD

**Discussion by dawn-breaking** [Vector35/binaryninja-api#5656] [General] [answered]
bininninjia commercial stable latest version save AwemeCore.bndb crashes on completion of analysis。latest commercial dev version load AwemeCore also crashed。example binary file download URL：https://dawnvip.oss-cn-shenzhen.aliyuncs.com/AwemeCore. os version : ubuntu22.04, memory total size 256G, with “full” for analysis mode

**Comment by emesare** [comment, marked as answer]
I went ahead and created an issue for you here: https://github.com/Vector35/binaryninja-api/issues/5657
